kong/plugins/acl/api.lua

```diff
@@ -53,5 +53,50 @@ return {
     DELETE = function(self, dao_factory)
       crud.delete(self.acl, dao_factory.acls)
     end
+  },
+
+  ["/acls"] = {
+    GET = function(self, dao_factory)
+      crud.paginated_set(self, dao_factory.acls)
+    end,
+
+    PUT = function(self, dao_factory)
+      crud.put(self.params, dao_factory.acls)
+    end,
+
+    POST = function(self, dao_factory)
+      crud.post(self.params, dao_factory.acls)
+    end
+  },
+
+  ["/acls/:id"] = {
+    before = function(self, dao_factory, helpers)
+      local filter_keys = {
+        id = self.params.id,
+        consumer_id = self.params.consumer_id,
+      }
+      self.params.group_or_id = nil
+
+      local acls, err = dao_factory.acls:find_all(filter_keys)
+      if err then
+        return helpers.yield_error(err)
+      elseif next(acls) == nil then
```

Comment:
should this be `elseif #acls == 0`, to match existing style, and as a more performant/idiomatic approach?